src/pages/home/report/ReportActionCompose.js

```diff
@@ -131,20 +132,24 @@ class ReportActionCompose extends React.Component {
 
         this.comment = props.comment;
         this.shouldFocusInputOnScreenFocus = canFocusInputOnScreenFocus();
+        this.selection = {
+            start: props.comment.length,
+            end: props.comment.length,
+        };
 
         this.state = {
             isFocused: this.shouldFocusInputOnScreenFocus,
             isFullComposerAvailable: props.isComposerFullSize,
             textInputShouldClear: false,
             isCommentEmpty: props.comment.length === 0,
             isMenuVisible: false,
-            selection: {
-                start: props.comment.length,
-                end: props.comment.length,
-            },
             maxLines: props.isSmallScreenWidth ? CONST.COMPOSER.MAX_LINES_SMALL_SCREEN : CONST.COMPOSER.MAX_LINES,
-            value: props.comment,
             conciergePlaceholderRandomIndex: _.random(this.props.translate('reportActionCompose.conciergePlaceholderOptions').length - 1),
+
+            // if this is undefined it means we haven't exceeded the max comment length
+            // if it is a number it means we have exceeded the max comment length and the number is the total length
+            // we only want to set this value when necessary to avoid re-renders
```

Comment:
```suggestion
            // If this is undefined it means we haven't exceeded the max comment length.
            // If it is a number it means we have exceeded the max comment length and the number is the total length.
            // We only want to set this value when necessary to avoid re-renders.
```